src/current/v23.1/monitoring-and-alerting.md

```diff
@@ -189,6 +189,685 @@ If you rely on external tools for storing and visualizing your cluster's time-se
 
 When storage of time-series metrics is disabled, the DB Console Metrics dashboards in the DB Console are still available, but their visualizations are blank. This is because the dashboards rely on data that is no longer available.
 
+### Critical nodes endpoint
+
+The critical nodes status endpoint is used to:
+
+- Check if any of your nodes are critical.  A node is 'critical' if that node becoming unreachable would cause [replicas to become unavailable](ui-cluster-overview-page.html#replication-status).
```

Comment:
Nit: This description is perfectly understandable, but it's somewhat vague whether "critical" is a state that a node is in (can a node go in and out of being critical?), or designation that it has (can a node critical 4 life?). I'd blindly assume the former, in which case I'd suggest something like "Check if any of your nodes are in a critical state".

(Italicizing per our style on introducing new terms)

```suggestion
- Check if any of your nodes are critical. A node is _critical_ if that node becoming unreachable would cause [replicas to become unavailable](ui-cluster-overview-page.html#replication-status).
```

Reply:
updated with your suggestion re: "in a critical state", and used the italics - thanks!